spec/classes/agent_spec.rb

```diff
@@ -160,4 +161,76 @@
       it { should contain_file('/etc/zabbix/zabbix_agentd.conf').with_content %r{^TLSPSKFile=/etc/zabbix/keys/tlspskfile.key$} }
     end
   end
+
+  context 'On Ubuntu' do
+    context 'trusty' do
+      let :facts do
```

Comment:
I only start involve with puppet and open source (only learned rspec last month), so i am still not familiar with the ruby and rspec (PHP background btw).

i can implement the changes, but it will be great if you can show me any reference or guideline that i can follow. as most of the stuff i pick up by reading the github source code directly (it is a bit hard to find the documentation on this)